woocommerce/woocommerce-android
# Activate: Emphasize credentials login for self-hosted sites

This PR implements the experiment in which we show the username/password screen for self-hosted sites instead of email login.

**To test:**
1. Start the app and tap on the "Enter your store address" button
2. Enter the address of a WP.com site
3. **After the site check, notice the login with email screen is shown**
4. Now go back to the site address screen
5. Enter the address of a non-WP.com site
6. **After the site check, notice the credentials login screen is shown**
